modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/RegistrationActivity.java

```diff
@@ -198,7 +184,8 @@ public void onReceiveAPIResult(Map<String, String> result, int requestCode) {
 				responseStatus = result.get(Constants.STATUS);
 				Preference.putString(context, resources.getString(R.string.shared_pref_regId), info.getDeviceId());
 				if (Constants.Status.SUCCESSFUL.equals(responseStatus)) {
-					if(Preference.getString(context, context.getResources().getString(R.string.shared_pref_notifier)).trim().equals(Constants.NOTIFIER_GCM)){
+					if(Preference.getString(context, context.getResources().
+							getString(R.string.shared_pref_notifier)).trim().equals(Constants.NOTIFIER_GCM)){
 						registerGCM();
 					}else{
```

Comment:
There seems to be a formatting issue in this particular line. Even though it doesnt appear to be something you'd added, might be good to fix it.